sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CommunicationUserIdentifier.java

```diff
@@ -15,7 +15,7 @@
     /**
      * Creates a CommunicationUserIdentifier object
      *
-     * @param id the string identifier representing the identity
+     * @param id id of the communication user.
```

Comment:
>id id of

➡ "id of"

Reply:
The first "id" is the parameter name. What about "@param id identifier string of the Communication user"

Reply:
oh I had misunderstood how the comments work in Java. I think we are good.